python/src/iceberg/utils/datetime.py

```diff
@@ -17,7 +17,9 @@
 """Helper methods for working with date/time representations
 """
 import re
-from datetime import date, datetime, time
+from datetime import date, datetime, time, timedelta
+
+import pytz
```

Comment:
I don't think this is needed. Instead of using pytz, this should convert using the python built-ins.